python/examples/rdm_snapshot.py

```diff
@@ -419,9 +419,9 @@ def _RDMRequestComplete(self, response, unpacked_data, unpack_exception):
 
 
 def Usage():
+  print("Usage: rdm_snapshot.py --universe <universe> [--input <file>] "
+        "[--output <file>]\n")
   print(textwrap.dedent("""\
-  Usage: rdm_snapshot.py --universe <universe> [--input <file>] [--output <file>]
```

Comment:
We could use the backslash here instead to avoid a separate print line if you'd rather?